pkg/apiserver/storage/vpc_inventory_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage_test
+
+/*var _ = Describe("Storage  test", func() {
```

Comment:
Why is this commented?? And dont name package as which includes _.. Also, why you testing storage??

Reply:
by error, deleted